packages/resource-detail/src/resource-detail.tsx

```diff
@@ -296,8 +301,45 @@ function ResourceDetail({
     enableOnOffSwitching: resourceOverviewMapWidget?.enableOnOffSwitching || false,
   } : {};
 
+  const GroupButtonDeleteEdit = () => (
+    <ButtonGroup>
+      { (displayDeleteButton && canDelete) && (
```

Comment:
Can you reverse these variables to be consistent with the other check?
```suggestion
      { (canDelete && displayDeleteButton) && (
```